JSTutorial/hw_8/task_4/Manager.js

```diff
@@ -0,0 +1,10 @@
+class Manager extends ITSpecialist{
+    #isScrumMaster
+    constructor() {
```

Comment:
Constructor of class should have at least the same arguments as a parent class + it's own. And you should use arguments, related to parent class, in super(parent class arguments)